### PR TITLE
Use dart-sass

### PR DESCRIPTION
**Pull request by zvizvi**
Use dart-sass, For compatibility with newer Nodejs versions

### CODE REVIEW

**Comment by nikitaeverywhere** [comment]
Hello! Thanks for contribution.

I believe this repo rather requires a full deps upgrade (to node 20+) or nvm'ing back to 14 or 12 (I don't remember which exactly).

If you run `npm run build` on your PR, it errors.

image

**Comment by nikitaeverywhere** [comment]
Plus as far as I can see dart-sass has slightly different syntax than scss (it tells about math.div?). I'm using `sass` in all my FE projects and so far it's OK (though that guys always struggle with backward compatibility which is demonstrated here)

**Comment by zvizvi** [comment]
Hi.
I updated Webpack to version 5.64.4 to support Node 20.
I also used the `math.div` method as required by the Dart Sass.
Thanks.

File: package.json [hunk 1]
```diff
@@ -36,12 +36,13 @@
...
     "dom-loaded": "^1.0.1",
     "gulp": "^4.0.0",
-    "gulp-sass": "^4.0.2",
+    "gulp-dart-sass": "^1.1.0",
     "js-untar": "^0.1.5",
     "querystringify": "^2.0.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
-    "webpack": "^5.10.0",
+    "sass": "^1.72.0",
```
Comment (nikitaeverywhere) [inline package.json]: Do we need this now?

File: yarn.lock
Comment (nikitaeverywhere) [inline yarn.lock:1]: I'm not using yarn anymore in my projects; if you don't use it either, we can drop this file and just use `package-lock.json`

**Review by nikitaeverywhere — APPROVED** [review]
Looks good; a few minor comments and we can merge, then I'll rebuild `docs`

File: yarn.lock
Comment (zvizvi) [reply yarn.lock:1]: I cleaned the unnecessary dependencies and changed to package.json
It works fine now.

**Comment by nikitaeverywhere** [comment]
Thank you a lot for your contribution.

**Comment by nikitaeverywhere** [comment]
Although, the project itself gets slightly old also in the light of npm introducing code preview feature

image

**Pull request merged**